jfrog-cli/artifactory/utils/golang/cmd.go

```diff
@@ -79,5 +81,37 @@ func RunGo(goArg string) error {
 	if err != nil {
 		return errorutils.CheckError(err)
 	}
+
+	regExp, err := utils.GetRegExp("((http|https)://)")
```

Comment:
For performance reasons, let's compile the regexp only once. This improvement will be relevant in cases when we run multiple go commands.

Reply:
It's being compiled only once. When running Go Cmd.